webapp/components/github-login.js

```diff
@@ -55,13 +67,19 @@ class GitHubLogin extends PolymerElement {
         type: String,
         value: null,
       },
+      isTriagedMode: {
```

Comment:
Note to reviewers: I haven't decided what to do here. Ideally I want to make it work like a client-side flag in flags/.